Removed reference to Ibiza, grammar and name fixes

kfile opened the pull request:
Ibiza is an old name for Azure portal. The link was correct, just the name was wrong.

azurecla commented:
Hi __@deneha__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

kfile committed:
Removed reference to Ibiza, grammar and name fixes

Ibiza is an old name for Azure portal. The link was correct, just the name was wrong.
